docker-compose.yml

```
@@ -4,10 +4,10 @@ x-logging: &logging
     max-file: 10
 
 x-image-main: &image-main
-  image: ${SHUTTER_IMAGE_ROLLING_SHUTTER:-ghcr.io/shutter-network/keyper:v1.3.12}
+  image: ${SHUTTER_IMAGE_ROLLING_SHUTTER:-ghcr.io/shutter-network/keyper:v1.3.13}
 
 x-image-assets: &image-assets
-  image: ${SHUTTER_IMAGE_ASSETS:-ghcr.io/shutter-network/assets:shutter-api-gnosis-1000-v0.0.1}
+  image: ${SHUTTER_IMAGE_ASSETS:-ghcr.io/shutter-network/assets:shutter-api-gnosis-1001-v1.1.1}
```

It's `1.0.1` not `1.1.1`
```suggestion
  image: ${SHUTTER_IMAGE_ASSETS:-ghcr.io/shutter-network/assets:shutter-api-gnosis-1001-v1.0.1}
```

---

The latest one is actually 1.1.1: https://github.com/orgs/shutter-network/packages/container/assets/562523242?tag=shutter-api-gnosis-1001-set1.1.1